tests/test_visitors/test_ast/test_statements/test_statements_with_no_effect.py

```diff
@@ -226,6 +249,9 @@ def test_statement_with_no_effect(
     'object.mro()',
     'del some',
     'some_var: int',
+    'x += 2',
```

Comment:
We also need to test: 
- `-=`
- `*=`
- `**=`
- `/=`
- `^=`
- `%=`
- `>>=`
- `<<=`
- `&=`
- `|=`

Reply:
added more tests